wfsim/core/s2.py

```diff
@@ -189,15 +189,27 @@ def get_s2_light_yield(positions, config, resource):
         
         returns array of floats (mean expectation) 
         """
-        sc_gain = resource.s2_correction_map(positions)
+        
+        if ('se_gain_from_map' not in config):
+            sc_gain = resource.se_gain_map(positions)
+        else:
+            if config['se_gain_from_map'] == False:
+                sc_gain = resource.se_gain_map(positions)
+            sc_gain = resource.se_gain_map(positions)
```

Comment:
in this conditional all conditions return the same things, don't they?
also, what is `sc` stand for? this is more for me to understand

Reply:
I guess it should be;
```python
       # calculate it from MC pattern map directly if no "se_gain_map" is given 
        if ('se_gain_from_map' not in config):
            sc_gain = resource.s2_correction_map(positions)
        else:
            if config['se_gain_from_map'] == False:
                sc_gain = resource.s2_correction_map(positions)
            else:
                sc_gain = resource.se_gain_map(positions)
```